docs/dev_guide/maintainer_workflow.rst

```diff
@@ -187,7 +187,7 @@ Updating and Maintaining the Changelog
 
 The changelog will be read by users, so this description should be aimed at SunPy users instead of describing internal changes which are only relevant to the developers.
 
-Instructions on how to write a changelog is given `here`_
+Instructions on how to write a changelog can be found `here`_
```

Comment:
Now that we are with this... we shouldn't use `here` on any of our links!
Read why ['click here' links are not good for accessibility on this Berkeley guide](https://webaccess.berkeley.edu/ask-pecan/click-here).

Reply:
How about now?